test/VestedPayment.js

```diff
@@ -0,0 +1,248 @@
+    await increaseTimeTestRPCTo(startTime+duration.days(300));
+
+    tokensAvailable = tokensAvailable.plus(help.lif2LifWei(30));
+
+    tokensAvailable.should.be.bignumber
+      .equal(await vestedPayment.getAvailableTokens.call());
+
+    await vestedPayment.claimTokens(help.lif2LifWei(10), {from: accounts[2]});
+    tokensAvailable = tokensAvailable.minus(help.lif2LifWei(10));
+
+    tokensAvailable.should.be.bignumber
+      .equal(await vestedPayment.getAvailableTokens.call());
+
+    // Go to period 11 plus 90 days and claim teh remaining tokens
+    await increaseTimeTestRPCTo(startTime+duration.days(320)+duration.days(90));
+
+    tokensAvailable = tokensAvailable.plus(help.lif2LifWei(5));
+
+    tokensAvailable.should.be.bignumber
+      .equal(await vestedPayment.getAvailableTokens.call());
+
+    await vestedPayment.claimTokens(tokensAvailable, {from: accounts[2]});
+
+    help.lif2LifWei(60).should.be.bignumber
+      .equal(await token.balanceOf(accounts[2]));
+
+  });
+
+
+  it(" should fail when try to claim tokens inside cliff periods", async function() {
+    const startTime = latestTime() + duration.days(60);
+    var vestedPayment = await VestedPayment.new(
+      startTime, duration.days(30), 12, 4, token.address
+    , {from: accounts[1]});
+    var tokensAvailable = new BigNumber(0);
+
+    await token.approve(vestedPayment.address, help.lif2LifWei(61), {from: accounts[1]});
+    await vestedPayment.fund(help.lif2LifWei(60), {from: accounts[1]});
+    await vestedPayment.transferOwnership(accounts[2], {from: accounts[1]});
+
+    assert.equal(accounts[2], await vestedPayment.owner.call());
+    assert.equal(true, await vestedPayment.funded.call());
+
+    // Go to period 1
+    await increaseTimeTestRPCTo(startTime+duration.days(30));
+
+    try {
+      await vestedPayment.claimTokens(help.lif2LifWei(5), {from: accounts[2]});
+    } catch (error) {
+      if (error.message.search('invalid opcode') == -1) throw error;
+    }
+
+  });
+
+  it(" should fail when try to claim more tokens than the available", async function() {
+    const startTime = latestTime() + duration.days(60);
+    var vestedPayment = await VestedPayment.new(
+      startTime, duration.days(30), 12, 0, token.address
+    , {from: accounts[1]});
+    var tokensAvailable = new BigNumber(0);
+
+    await token.approve(vestedPayment.address, help.lif2LifWei(61), {from: accounts[1]});
+    await vestedPayment.fund(help.lif2LifWei(60), {from: accounts[1]});
+    await vestedPayment.transferOwnership(accounts[2], {from: accounts[1]});
+
+    assert.equal(accounts[2], await vestedPayment.owner.call());
+    assert.equal(true, await vestedPayment.funded.call());
+
+    // Go to period 1
+    await increaseTimeTestRPCTo(startTime+duration.days(30));
+
+    try {
+      await vestedPayment.claimTokens(help.lif2LifWei(15), {from: accounts[2]});
+    } catch (error) {
+      if (error.message.search('invalid opcode') == -1) throw error;
```

Comment:
wanna replace this with `if (!help.isInvalidOpcodeEx(error)) throw error`? same for other occurrences in this file

Reply:
:+1: